lib/src/base-styles/_nx-colors.scss

```diff
@@ -677,14 +677,16 @@
     --nx-color-page-title-divider: var(--nx-swatch-indigo-25);
   }
 
-  // Deprecated variables
-  --nx-color-subsection-border: var(--nx-color-border);
-  --nx-color-text-dark: var(--nx-color-text-stark);
-  --nx-color-form-element-border: var(--nx-swatch-grey-50); // form - element - border
-  --nx-color-interactive-border-pristine: var(--nx-swatch-indigo-90);
-  --nx-color-interactive-shadow-focus: hsla(var(--nx-swatch-teal-hs), 35%, 0.5);
-  --nx-color-alert-warning: var(--nx-swatch-orange-50);
-  --nx-color-alert-info: var(--nx-swatch-blue-40);
-  --nx-color-alert-success: var(--nx-swatch-green-25);
-  --nx-color-alert-error: var(--nx-swatch-red-40);
+  & {
```

Comment:
I wanted these to stay at the bottom of the file, so I used this syntax instead of moving them.